__tests__/app.test.js

```diff
@@ -83,3 +83,35 @@ describe('GET /api/articles/:article_id',() => {
     })
 
 })
+describe('GET /api/articles', () => {
+    test('status 200, will return an array of articles containing the correct properties ', ()=> {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body }) => {
+            const { articles } = body
+            const regEx = /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{3}Z$/g
+            expect(articles).toHaveLength(13)
+            articles.forEach((article) => {
+                expect(article).toHaveProperty('author', expect.any(String));
+                expect(article).toHaveProperty('title', expect.any(String));
+                expect(article).toHaveProperty('article_id', expect.any(Number));
+                expect(article).toHaveProperty('topic', expect.any(String));
+                expect(article).toHaveProperty('created_at', expect.stringMatching(regEx));
+                expect(article).toHaveProperty('votes', expect.any(Number));
+                expect(article).toHaveProperty('article_img_url',expect.any(String));
+                expect(article).toHaveProperty('comment_count', expect.any(String));
```

Comment:
You will have to deal with this being a string in the front end. This is no huge deal but you can nadle it with SQL...

https://www.w3schools.com/sql/func_sqlserver_cast.asp